warehouse/internal/model/upload.go

```diff
@@ -36,7 +36,10 @@ const (
 	Noop                      JobErrorType = "noop"
 )
 
-var ErrUploadNotFound = errors.New("upload not found")
+var (
+	ErrUploadNotFound = errors.New("upload not found")
+	ErrNoUploadsFound = errors.New("no uploads found")
```

Comment:
[question] Isn't this the same error? Why do we need to have both ?

Reply:
I thought `ErrUploadNotFound` was only for error when we looked for a particular upload. In my case, I was looking for uploads for a particular source and destination.

Reply:
Should I use sql.ErrNoRows instead of sentinal error?